Sources/Sentry/SentryProfiler.mm

```diff
@@ -168,6 +168,19 @@ - (instancetype)init
 
 + (void)startForSpanID:(SentrySpanId *)spanID hub:(SentryHub *)hub
 {
+#    if SENTRY_TARGET_PROFILING_SUPPORTED
+    NSTimeInterval timeoutInterval = 30;
+#        if defined(TEST) || defined(TESTCI)
+    timeoutInterval = 1;
+#        endif
```

Comment:
I never thought about this hack 😁. Very pragmatic 👍

Reply:
Thanks, quick n dirty saves the day 😄 If it comes back to bite us, lets revisit. But, we're going to have continuous profiling at some point, so this will probably get a substantial refactor again.